manifests/user-roles.jsonnet

```diff
@@ -16,6 +17,11 @@ local kubecfg = import "kubecfg.libsonnet";
           resources: ["*"],
           verbs: ["list", "get", "watch"],
         },
+        {
+          apiGroups: ["*"],
+          resources: ["selfsubjectrulesreviews", "selfsubjectaccessreviews"],
```

Comment:
AFAIK this is not required, you'd only need this if you were using SubjectRulesReviews/SubjectAccessReviews. A user is always allowed to do a SelfSubject*Review

Reply:
That was the case when I was checking Minikube (1.9) but in GKE (1.8) it wasn't (probably it's a bug on their side?). I needed to add these permissions for it to work.